documentation/Using-the-SDK/Building-a-table.md

```diff
@@ -1,21 +1,27 @@
 # Building a Table
 
-Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add both Columns and `TableConfigurations`.
+Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add `TableConfigurations` and to establish a row count for the table.
 
-* [Column](#column)
-  * [ColumnConfiguration](#columnconfiguration)
-  * [Projection](#projection)
-  * [Combining ColumnConfiguration and Projections](#combining-columnconfiguration-and-projections)
-* [TableConfiguration](#tableconfiguration)
-  * [ColumnRole](#columnrole)
+*Note*: Every projection in the table must support the table's established row count.
+
+Setting the table's row count (`ITableBuilder.SetRowCount`) returns an instance of `ITableBuilderWithRowCount` which has a method to add columns to the table.
+
+- [Building a Table](#building-a-table)
+  - [Column](#column)
+    - [ColumnConfiguration](#columnconfiguration)
+    - [Projection](#projection)
+    - [Combining ColumnConfiguration and Projections](#combining-columnconfiguration-and-projections)
+  - [TableConfiguration](#tableconfiguration)
+    - [ColumnRole](#columnrole)
+- [More Information](#more-information)
 
 ## Column
 
-A column is a conceptual (`ColumnConfiguration`, `Projection`) pair that defines data inside a [Table](#table).
+A column is a conceptual pair (`ColumnConfiguration`, `Projection`) that defines data inside a [Table](#table).
```

Comment:
Can you remove the Table link? It was copied from the glossary by mistake.

Reply:
sure